packages/prisma/schema.prisma

```diff
@@ -72,18 +73,17 @@ enum SubscriptionStatus {
 model Subscription {
   id                Int                @id @default(autoincrement())
   status            SubscriptionStatus @default(INACTIVE)
-  planId            String?
-  priceId           String?
+  planId            String             @unique
+  priceId           String
   customerId        String
```

Comment:
Should we be tossing the customerId here since it's on the user?

Reply:
That's true, I've removed it but it required some changes to the webhook handlers

Also noticed that we aren't actually updating `cancelAtPeriodEnd` when a subscription changes so I've included those changes